.github/ISSUE_TEMPLATE/NEW_FEATURE.md

```diff
@@ -0,0 +1,39 @@
+---
+name: New feature
+about: Create a request for new functionality in devdocs
+---
+
+# New feature request
+
+## Description
+
+<!-- (REQUIRED) Describe the feature you want added to devdocs -->
+
+## Expected result
+
+<!-- (REQUIRED) When this feature is implemented, what is the expected result/behavior? -->
```

Comment:
<del>When this feature is implemented,</del> What is the expected result/behavior of this feature?